src/Menu.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'item'
+#require 'twilio-ruby'
+
+class Menu
+    attr_reader :basket, :menu
+
+    def initialize()
+        @basket = []
+        @menu = [Item.new("Pizza", 3.99), Item.new("Burger", 4.99), Item.new("Fries", 2.99)]
+    end
+
+    def AddToBasket(itemName, quantity)
+        @menu.each do |item|
+            if item.name == itemName
+                quantity.times do
+                    @basket << item
+                end
+            end
+        end
+    end
+
+    def GetTotal
+        total = 0
+        @basket.each do |item|
+            total += item.price
+        end
+        return total
+    end
+
+    def PurchaseOrder
+=begin
+        @client = Twilio::REST::Client.new(
+            ENV['ACCOUNT_SID'],
```

Comment:
This looks like you are on the right track. Would be hard to test this method without actually sending the message. Is there a way we could design this class so that we can check the message sent back to the user without actually sending a text?